bitnami/influxdb/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Database
 apiVersion: v2
-appVersion: 2.0.9
+appVersion: 2.1.1
```

Comment:
This shouldn't be updated as the chart is running 2.0.9 at the moment.

Reply:
Can this be updated as 2.1.1 is released? I had opened up an issue in bitnami-docker-influxdb repo but it was closed as my description was not liked. In any case, I was hoping to get the latest version in with this pr itself and was waiting for someone to push that image.

Reply:
The chart will be automatically updated when the image is released (our team is working on that, it has to pass some internal testing before it is published). Since the process is automatic, there's no point on waiting for this until 2.1.1 is realesed.